docs/reference/modules/node.asciidoc

```diff
@@ -360,10 +360,14 @@ To create a dedicated {ml} node, set:
 node.roles: [ ml, remote_cluster_client] <1>
 xpack.ml.enabled: true <2>
 ----
-<1> The `remote_cluster_client` role is optional but strongly recommended.
+<1> The `remote_cluster_client` role is optional but strongly recommended on all {ml} nodes.
 Otherwise, {ccs} fails when used in {ml} jobs or {dfeeds}. See <<remote-node>>.
 <2> The `xpack.ml.enabled` setting is enabled by default.
 
+WARNING: The `remote_cluster_client` role is required on all master-eligible nodes in case
```

Comment:
```suggestion
WARNING: If you use {ccs} in your {anomaly-jobs}, the `remote_cluster_client` role is required on all master-eligible nodes.
```

Reply:
correct, only if using ccs. We will only make the check if a ccs index is included in the datafeed.